.github/workflows/python-ci.yml

```diff
@@ -33,6 +31,7 @@ jobs:
   unit:
     # 1st condition: push event (restricted to master above)
     # 2nd condition: PR opened/synchronized/reopened (types filter above)
+    #                except Release PRs, those need a manual CI:TEST
```

Comment:
I'll come up with an automated solution that will run CI on a Release PR just prior to merging it.